source/guides/clear/stateless.rst

```diff
@@ -36,18 +36,20 @@ between user-owned areas and |CL|-owned areas.
 
 System areas
 ============
-File under the :file:`/usr` directory are managed by |CL| as system files.
+File under the :file:`/usr` directory are managed by |CL| as system files
```

Comment:
This was an issue in the previous revision, but the first word of this sentence should be `Files`.